proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -50,7 +50,7 @@ message MsgTransfer {
   // optional memo
   string memo = 8;
   // tokens to be transferred
-  repeated cosmos.base.v1beta1.Coin tokens = 9 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
```

Comment:
it doesn't seem to have any effect because for a single token amount always shows as a `"0"` string because the amount field is a custom into sdkmath.Int. 

@chatton and I tested a bunch of different variations this morning.